### ISSUE TITLE
HW Acceleration and OpenVPN

### ISSUE THREAD

**Issue by adizanni** [geekman/xqrepack#28]
Sorry, this is more a request for information than a real issue.
I was wondering if this modded firmware preserves the Crypto HW accelerations and allow the installation of OpenVPN ?

**Comment by geekman** [comment]
Yes it keeps the hw acceleration, if it even has that. It doesn't directly enable installation of software packages, but you probably can drop a binary somewhere when you SSH into the router.
